### PR TITLE
bug(TMC-22935) Can't set schedule to task,got " Failed to save schedule." 504 error

### PR DESCRIPTION
**Pull request by ozatsepin**
**What is the problem this Pull Request is trying to solve?**
Requests to TMC WEB APP fails when X-Forwarded-Proto header is wrong
 
**What is the chosen solution to this problem?**
add aditional validation for scheme 

**Link to the JIRA issue**
https://jira.talendforge.org/browse/TMC-22935
 
**Please check if the Pull Request fulfills these requirements**
- [ ] The PR commit message follows our [guidelines](https://github.com/Talend/daikon/blob/master/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features, coverage should be over 75% in the new code)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in JIRA), if any, are all linked or available in the Pull Request

 
**[ ] This Pull Request introduces a breaking change**

### CODE REVIEW

File: daikon-spring/daikon-spring-audit-logs/src/main/java/org/talend/daikon/spring/audit/logs/service/AuditLogContextBuilder.java [hunk 1]
```diff
@@ -230,7 +230,9 @@ private void computeRequestFields() {
     private String computeRequestUrl(HttpServletRequest httpServletRequest) {
         if (!StringUtils.isEmpty(httpServletRequest.getHeader("X-Forwarded-Host"))) {
             return UriComponentsBuilder.fromPath(httpServletRequest.getRequestURI())
-                    .scheme(Optional.ofNullable(httpServletRequest.getHeader("X-Forwarded-Proto")).orElse("https"))
+                    .scheme(Optional.ofNullable(httpServletRequest.getHeader("X-Forwarded-Proto"))
```
Comment (fhuaulme) [inline daikon-spring/daikon-spring-audit-logs/src/main/java/org/talend/daikon/spring/audit/logs/service/AuditLogContextBuilder.java:233]: Please run a `mvn clean install` using `-Dformat` option to make sure code format is ok.

**Review by fhuaulme — CHANGES_REQUESTED** [review]

File: daikon-spring/daikon-spring-audit-logs/src/main/java/org/talend/daikon/spring/audit/logs/service/AuditLogContextBuilder.java (hunk 1 above)
Comment (ozatsepin) [reply daikon-spring/daikon-spring-audit-logs/src/main/java/org/talend/daikon/spring/audit/logs/service/AuditLogContextBuilder.java:233]: built successful

Comment (fhuaulme) [reply daikon-spring/daikon-spring-audit-logs/src/main/java/org/talend/daikon/spring/audit/logs/service/AuditLogContextBuilder.java:233]: Yes, but did it build *and* you don't have local changes to push to GitHub?

Comment (ozatsepin) [reply daikon-spring/daikon-spring-audit-logs/src/main/java/org/talend/daikon/spring/audit/logs/service/AuditLogContextBuilder.java:233]: yes, also it was built on CI already

**Review by fhuaulme — APPROVED** [review]

**Pull request merged**